secator/runners/_base.py

```
@@ -284,9 +288,54 @@ def _process_config(self, config):
+
+		collect(self)
+		try:
+			from secator.runners.task import Task
+			from secator.tree import build_runner_tree, get_flat_node_list
+			for node in get_flat_node_list(build_runner_tree(self.config)):
+				if node.type != 'task':
+					continue
+				try:
+					collect(Task.get_task_class(node.name))
+				except Exception:
+					continue
```

Ruff (0.15.20)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)